src/ui/windows/TogglDesktop/TogglDesktop/AutoCompletion/ListBoxItem.cs

```diff
@@ -16,7 +17,18 @@ class ListBoxItem
         public string TaskLabel { get; set; }
         public string ClientLabel { get; set; }
         public string Category { get; set; }
-        public string WorkspaceName { get; set; }
+        private string _workspaceName = "";
+        public string WorkspaceName
+        {
+            get
+            {
+                return this._workspaceName;
+            }
+            set
+            {
+                this._workspaceName = CultureInfo.CurrentCulture.TextInfo.ToTitleCase(value);

```

Comment:
`ToTitleCase` capitalizes only the first letters of words, e.g. `project name` -> `Project Name`. We need to use `string.ToUpper()` or `string.ToUpperInvariant()` method here.